openssl/src/crypto/rsa.rs

```diff
@@ -7,7 +7,7 @@ use std::io::{self, Read};
 use bn::BigNum;
 use bio::MemBio;
 
-pub struct RSA(*mut ffi::RSA);
+pub struct RSA(pub *mut ffi::RSA);
```

Comment:
you can see the usage on pkey.rs:228 above. I can keep that private but add a public constructor like with_raw(..) if you'd prefer? Your call.